scripts/testnet/include/observers.sh

```diff
@@ -86,7 +86,7 @@ assembleCommand_startObserverNode() {
         -port $PORT --profile-mode -log-save -log-level $LOGLEVEL --log-logger-name --log-correlation --use-health-service -rest-api-interface localhost:$RESTAPIPORT \
         -destination-shard-as-observer $SHARD \
         -sk-index $KEY_INDEX \
-        -working-directory $WORKING_DIR -config ./config/config_observer.toml"
+        -working-directory $WORKING_DIR -config ./config/config_observer.toml -full-archive"
```

Comment:
maybe add a flag inside local testnet for tagging a node as full archive?

Reply:
also for consistency please refactor to `--multiple-words` instead of `-multiple-words`

Reply:
Will put the switch but it will default to true as to test as much as we can this feature

Reply:
done